src/main/webapp/WEB-INF/jsp/expiredActivationLink.jsp

```diff
@@ -16,13 +16,12 @@
             <spring:message code="public.user.activation.expired.text.p1.1"/>
             <c:choose>
                 <c:when test="${empty redirect_uri}">
-                    <spring:message code="public.user.activation.expired.text.p1.2"/>
+                    <spring:message code="public.user.activation.expired.text.p1.2"/>.
```

Comment:
This period seems to belong to the message. Shouldn't it be added to respective `messages*.properties` files?